src/api/utils/getGoods.ts

```diff
@@ -0,0 +1,5 @@
+import { API_URL } from '../const/constants';
+/* import { getData } from './getData'; */
+
+export const getGoods = (): Promise<Good[]> => (fetch(API_URL).then(response => response.json()));
```

Comment:
unneeded parenthesis
```suggestion
export const getGoods = (): Promise<Good[]> => fetch(API_URL).then(response => response.json());
```
or write multiline